compiler/rustc_mir_dataflow/src/framework/mod.rs

```diff
@@ -226,21 +226,19 @@ pub trait Analysis<'tcx>: AnalysisDomain<'tcx> {
     ///
     /// Unlike the other edge-specific effects, which are allowed to mutate `Self::Domain`
     /// directly, overriders of this method must pass a callback to
-    /// `SwitchIntEdgeEffects::apply`. The callback will be run once for each outgoing edge and
```

Comment:
The reason this is plural is that each `SwitchInt` can result in custom effects along multiple edges. `call_return` and `yield_resume` only apply along a single edge. Is there a reason to change this?

Reply:
There were seven `apply_..._effect`, but only one `apply_..._effects`, so I thought it was an oversight. But I don't feel strongly about it. I will change it back.